app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionDialog.kt

```diff
@@ -183,4 +185,16 @@ class BloodSugarUnitSelectionDialog : AppCompatDialogFragment(), BloodSugarUnitS
   interface BloodSugarUnitSelectionDialogInjector {
     fun inject(target: BloodSugarUnitSelectionDialog)
   }
+
+  @Parcelize
+  data class Key(
+      private val bloodSugarUnitPreference: BloodSugarUnitPreference,
+      override val type: ScreenType = ScreenType.Modal,
+      override val analyticsName: String = "Blood Sugar Unit Selection Dialog"
+  ) : ScreenKey() {
+
+    override fun instantiateFragment() = BloodSugarUnitSelectionDialog().apply {
+      isCancelable = false
```

Comment:
We use this to avoid dismissing when the user clicks outside the dialog. But it doesn't seem like existing dialog has that implementation. So, we can remove this?